src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -631,6 +639,44 @@ public void addEnvVars(List<TemplateEnvVar> envVars) {
         }
     }
 
+    /**
+     * Deletes the template.
+     */
+    @RequirePOST
```

Comment:
I see you're using `POST` in other places, is there any specific reason to use `RequirePOST`here?

Reply:
I will change it.